ClassificationNeuralNetwork.java

```diff
@@ -62,4 +82,41 @@ public static void readDataFromFile()
 					+ currLineNum);
 		}
 	}
+	
+	public static void putDataInArrayList()
+	{
+		for(int i=0; i<numLines; i++)
+		{
+			double[] toAdd= new double[10];
+			
+			toAdd[0]=radii[i];
+			toAdd[1]=textures[i];
+			toAdd[2]=perimeters[i];
+			toAdd[3]=area[i];
+			toAdd[4]=smoothness[i];
+			toAdd[5]=compactness[i];
+			toAdd[6]=concavity[i];
+			toAdd[7]=concavePnts[i];
+			toAdd[8]=symmetry[i];
+			toAdd[9]=fractalDim[i];
+			
+			listOfTestData.add(toAdd);
+			listOfTestClassifications.add(classification[i]);
+		}
+	}
+	
+	public static void shuffle()
```

Comment:
I suggest naming this something a little more descriptive as it does more than shuffle

Reply:
Done.